apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/index.tsx

```diff
@@ -192,79 +193,88 @@ const TransactionConfirm = ({
   const key = ("mode" in transaction && transaction.mode) || "send";
   const recipientWording = t(`TransactionConfirm.recipientWording.${key}`);
 
-  return typeTransaction === "Approve" ? (
-    <Container style={{ paddingBottom: 0 }}>
-      <Container paddingX={26}>
-        <DeviceBlocker />
-        <Animation animation={getDeviceAnimation(device.modelId, type, "verify")} />
-        <Text ff={"Inter|Medium"} textAlign={"center"} fontSize={22} marginBottom={12}>
-          {t("approve.description")}
-        </Text>
-        <Alert type="primary" mb={26}>
-          <Trans
-            i18nKey={
-              amountTransaction === "Unlimited MATIC" ? "approve.unlimited" : "approve.limited"
-            }
-            values={{
-              recipientWording,
+  console.log("typeTransaction", typeTransaction);
+
+  if (typeTransaction === "Approve" || typeTransaction === "Present")
+    return (
+      <Container style={{ paddingBottom: 0 }}>
+        <Container paddingX={26}>
+          <DeviceBlocker />
+          <Animation animation={getDeviceAnimation(device.modelId, type, "verify")} />
+          <Text ff={"Inter|Medium"} textAlign={"center"} fontSize={22} marginBottom={12}>
+            {typeTransaction === "Approve" ? t("approve.description") : t("sign.description")}
+          </Text>
+          <Alert type="primary" mb={26}>
+            <Trans
+              i18nKey={
+                typeTransaction === "Approve"
+                  ? amountTransaction === "Unlimited MATIC"
+                    ? "approve.unlimited"
+                    : "approve.limited"
+                  : "sign.securityCheck"
+              }
+              values={{
+                recipientWording,
+              }}
+            />
+          </Alert>
+          <Box
+            style={{
+              width: "100%",
             }}
-          />
-        </Alert>
-        <Box
-          style={{
-            width: "100%",
-          }}
-          mb={20}
-        >
-          {fields.map((field, i) => {
-            const MaybeComponent = fieldComponents[field.type];
-            if (!MaybeComponent) {
-              console.log(
-                `TransactionConfirm field ${field.type} is not implemented! add a generic implementation in components/TransactionConfirm.js or inside families/*/TransactionConfirmFields.js`,
+            mb={20}
+          >
+            {fields.map((field, i) => {
+              const MaybeComponent = fieldComponents[field.type];
+              if (!MaybeComponent) {
+                console.log(
```

Comment:
- Could you make this a console.warn?
- is it a case that should NOT happen? only in development phase? i would just not expect a production app to display these, if it's a normal case, then we don't want it to log

Reply:
I'd guess it's useful in development phase and later on to catch potential oversights. 
In any case for now I've turned it into a console.warn instead.